gateway/1.4/modules/ROOT/pages/flex-conn-reg-run-app.adoc

```diff
@@ -5,29 +5,29 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
 
-[cols="1a,1a,1a"]
-|===
-|image:install-linux-logo.png[20%,20%,xref="flex-conn-reg-run-app.adoc#register-and-run-with-a-connected-app-as-a-linux-service"]
-|image:install-docker-logo.png[25%,25%,xref="flex-conn-reg-run-app.adoc#register-and-run-with-a-connected-app-in-a-docker-container"]
-|image:install-kubernetes-logo.png[20%,20%,xref="flex-conn-reg-run-app.adoc#register-and-run-with-a-connected-app-as-a-kubernetes-ingress-controller"]
-
-|xref:flex-conn-reg-run-app.adoc#register-and-run-with-a-connected-app-as-a-linux-service[Register and Run with a Connected App as a Linux Service]
-|xref:flex-conn-reg-run-app.adoc#register-and-run-with-a-connected-app-in-a-docker-container[Register and Run with a Connected App in a Docker Container]
-|xref:flex-conn-reg-run-app.adoc#register-and-run-with-a-connected-app-as-a-kubernetes-ingress-controller[Register and Run With a Connected App as a Kubernetes Ingress Controller]
-|===
+//table with tech logos and links (linux, docker, k8, openshift)
+include::partial$task-reg-run-flex-gateway.adoc[tags=table-logos-links]
 
 // Prerequisites
 include::partial$task-reg-run-flex-gateway.adoc[tags=prerequisites-heading;app-prerequisites;prerequisites;environment-prerequisites]
 
+[[linux]]
 // Register and run with a connected app as a Linux service
 include::partial$task-reg-run-flex-gateway.adoc[tags=app-linux-heading;reg-command-intro;app-replace-content;environment-replace-content;replace-content;reg-command-heading;reg-command-1;linux-reg-command;app-reg-command;environment-reg-command;connected-reg-command;organization-reg-command;output-reg-command-linux;reg-command-2;after-reg;linux-after-reg;connected-after-reg;create-config-folder-file;config-content;linux-start-commands;gateway-connected]
 
+[[docker]]
 // Register and run with a connected app in a Docker container
 include::partial$task-reg-run-flex-gateway.adoc[tags=app-docker-heading;reg-command-intro;sub-coll-info;app-replace-content;environment-replace-content;replace-content;reg-command-heading;reg-command-1;docker-reg-command;app-reg-command;environment-reg-command;connected-reg-command;organization-reg-command;output-reg-command-docker;reg-command-2;after-reg;after-reg-2;connected-after-reg;start-command;gateway-connected]
 
+[[k8]]
 // Register and run with a connected app as a kubernetes ingress controller
 include::partial$task-reg-run-flex-gateway.adoc[tags=app-k8s-heading;k8s-reg-command-intro;sub-coll-info;app-replace-content;environment-replace-content;replace-content;reg-command-heading;reg-command-1;docker-reg-command;app-reg-command;environment-reg-command;connected-reg-command;organization-reg-command;output-reg-command-docker;reg-command-2;after-reg;k8s-after-reg;connected-after-reg;k8s-install-helm-chart;gateway-connected;helm-chart-options]
 
+[[openshift]]
```

Comment:
TODO: in every case, maybe we can say **generally** (without walking through the actual UI, which can change) how they could use OpenShift ([OpenShift Container Platform](https://docs.openshift.com/container-platform/4.12/welcome/index.html)?) to update the Helm chart and add the registration info as per https://docs.google.com/document/d/1C-J60TzEv9hiS59bGR83HreL2PoWlSWP7mdYZsjF0ng/edit#heading=h.p9eyep3bpinl. Need to discuss.